Makefile

```diff
@@ -164,7 +164,7 @@ build-dll: node_modules
 	@mkdir -p build
 	@CALYPSO_ENV=$(CALYPSO_ENV) $(NODE_BIN)/webpack --display-error-details --config webpack-dll.config.js
 
-build-server: install
+build-server: install server/devdocs/proptypes-index.json
```

Comment:
I realized I forgot to commit my tests, and the build failed without that file existing. I'd like to chat about it in slack if you don't mind. I don't like it.